tests/conftest.py

```diff
@@ -56,6 +56,17 @@ def unsupported_cve():
         return Document.from_data(data)
 
 
+@pytest.fixture
+def unsupported_cve_none_versions():
+    """Unsupported ecosystem CVE fixture.
+
+    Some of the CVE versions in this CVE are "None".
+    """
+    with open('tests/data/unsupported-nvdcve-2.json') as f:
+        data, = json.load(f)['CVE_Items']
```

Comment:
why do you use comma here? is the structure different than you expect?

Reply:
Hmm, let me check. I honestly don't remember. Maybe it is a typo? Copy pasted all over the file? :)

Reply:
It's an array, always with just one element (in this test data).